bindings/rust/src/bindings/mod.rs

```diff
@@ -210,7 +231,8 @@ impl KZGProof {
                 kzg_settings,
             );
             if let C_KZG_RET::C_KZG_OK = res {
-                Ok((kzg_proof.assume_init(), y_out.assume_init()))
+                let kzg_proof = KzgProof::from_binding_bytes(kzg_proof.assume_init());
```

Comment:
Hmm, I was surprised to see changes to the core bindings code. Also, these changes were completely hidden because of the directory restructuring in later commits.

Reply:
Late for this suggestion, but this is PR is intended to be read commit by commit since, as you can tell, the final diff is unhelpful. The generated bindings differ mostly because the current ones are heavily edited by hand

Reply:
Sounds good! Will review tomorrow more deeply.